lib/policy/checker.go

```diff
@@ -110,11 +110,11 @@ func NewUserAgentChecker(rexStr string) (Checker, error) {
 }
 
 func NewHeaderMatchesChecker(header, rexStr string) (Checker, error) {
-	rex, err := regexp.Compile(rexStr)
+	rex, err := regexp.Compile(strings.TrimSpace(rexStr))
```

Comment:
Even though it's not possible for Anubis' normal config loading route to end with trailing spaces here, be paranoid.